crates/net/network/benches/broadcast.rs

```diff
@@ -31,7 +31,8 @@ pub fn broadcast_ingress_bench(c: &mut Criterion) {
                 // runtime context and simply calling `block_on` here will cause the code to panic.
                 tokio::task::block_in_place(|| {
                     tokio::runtime::Handle::current().block_on(async {
-                        let provider = MockEthProvider::default();
+                        use reth_primitives::TransactionSigned;
+                        let provider = MockEthProvider::<TransactionSigned>::default();
```

Comment:
These changes stink but i think it's unavoidable, even though i hid the op impls behind the optimism feature flag

Reply:
eventually we want to remove those op feature flags anyway, I don't think it stinks so bad :)